docs/source/conf.py

```diff
@@ -181,7 +181,6 @@ def render_svg_logo(path):
     '_gallery_original',
     'generated-examples',
 ]
-myst_heading_anchors = 3
```

Comment:
Is this a intentional change?

Reply:
Yes, this overrides the `myst_heading_anchors = 7` defined previously and will make nested anchors in markdown not work